__tests__/apps.test.js

```diff
@@ -256,10 +306,36 @@ describe("app", () => {
                 expect(body.message).toBe("This is not a user")
             })
         })
+        test('400: Should return error for invalid sort query ', () => {
+            return request(app)
+            .get("/api/articles")
+            .query({sort_by: "inValidSortBy", order: "ASC", topic: "mitch"})
+            .expect(400)
+            .then(({body}) => {
+                expect(body.message).toBe("Invalid sort query")
+            })
+        });
+        test('400: Should return error for invalid order query ', () => {
+            return request(app)
+            .get("/api/articles")
+            .query({sort_by: "artile_id", order: "inValidOrder", topic: "mitch"})
```

Comment:
You have miss-spelled article_id here so it will be throwing your sort by error but needs to just have order with incorrect value and should throw value error